packages/angular/collection.json

```diff
@@ -115,6 +122,11 @@
       "aliases": ["app"],
       "description": "Creates an Angular application."
     },
+    "init-mfe": {
```

Comment:
We need to add this generator to the generators list in Angular Plugin overview docs.

Reply:
Does that not get done with the documentation hook? There's an init-mfe.md file in docs/angular/api-angular/generators.

Is there a different location needs updated?

Reply:
The documentation scripts will generate the pages for each generator and executor, but it won't generate the Plugin overview, that's an overview page that we manually maintain. I recently updated it because I was not aware of it either and it was outdated. You can see it here https://nx.dev/latest/angular/angular/overview#executors--builders and you can find the docs source in `docs/shared/angular-plugin.md`.

Reply:
Thanks for the clarification, I'll get this cleared up now!